scrapy_poet/utils.py

```diff
@@ -0,0 +1,15 @@
+from tldextract import tldextract
+
+
+def get_domain(url):
```

Comment:
It seems this will be called many times while crawling a website.
Maybe am alternative would be to use a parse cache
https://github.com/scrapy/scrapy/blob/e42d82526a1a519ff1305b189acde064a40fbd8d/scrapy/utils/httpobj.py#L10

Reply:
Yes, I would say this would even be a thing to be moved to Scrapy itself. It is pretty useful to be able to extract the domain. Here is only called twice per request, so I don't think it is worth it.